tests/plugins/test_add_labels.py

```diff
@@ -704,3 +704,75 @@ def test_add_labels_base_image(tmpdir, docker_tasker, parent, should_fail,
     else:
         runner.run()
         assert df.labels['release'] == '5'
+
+
+@pytest.mark.parametrize('base_new, df_new, plugin_new, expected_in_df, expected_log', [  # noqa
+    (None,  None,  None,  None,  None),
+    (None,  'A',   'A',   'A',   None),
+    (None,  'B',   'A',   'A',   'setting label'),
+    (None,  'A',   'B',   'B',   None),
+    (None,  None,  'A',   'A',   'setting label'),
+    (None,  'A',   None,  'A',   None),
+    ('A',   None,  'A',   'A',   'setting label'),
+    ('B',   None,  'A',   'A',   'setting label'),
+    ('A',   None,  'B',   'B',   'setting label'),
+    ('A',   'A',   None,  'A',   None),
+    ('A',   'B',   None,  'B',   None),
+    ('B',   'A',   None,  'A',   None),
+    ('A',   'A',   'A',   'A',   None),
+    ('A',   'B',   'A',   'A',   'setting label'),
+    ('B',   'A',   'A',   'A',   None),
+    ('A',   'A',   'B',   'B',   'setting label'),
+    ('A',   'B',   'B',   'B',   None),
+    ('B',   'B',   'A',   'A',   'setting label'),
+    ('B',   'A',   'B',   'B',   'setting label'),
+    ('A',   'B',   'C',   'C',   'setting label'),
+])
+def test_release_label(tmpdir, docker_tasker, caplog,
+                       base_new, df_new, plugin_new,
+                       expected_in_df, expected_log):
+    if MOCK:
+        mock_docker()
+
+    df_content = "FROM fedora\n"
+    plugin_labels = {}
+
+    if df_new:
+        df_content += 'LABEL release="{0}"\n'.format(df_new)
```

Comment:
I'm not sure why is `release` being tested there. Having `rewrite_whitelist` removed will this behaviour work on any label? I don't think that's what we want